mon_school/www/sketches/index.html

```diff
@@ -52,3 +52,30 @@
   </ul>
 </nav>
 {% endmacro %}
+
+{% block script %}
+
+  <script type="text/javascript">
+    function toggleLike(element) {
+      var sketchName = element.getAttribute("data-sketch-name");
+      frappe.call("mon_school.mon_school.doctype.lms_sketch_like.lms_sketch_like.toggle_like_sketch", {
+        sketch_name: sketchName,
+      })
+      .then(r => {
+        if (r.ok) {
+          window.location.reload();
```

Comment:
Reloading a page on liking a sketch is not a good idea. We need to figure out a way to update the count and status without reloading.

Reply:
Ah, you already have it mentioned in your todos.

Reply:
This is done